records_mover/records/prep_and_load.py

```diff
@@ -15,7 +15,7 @@ def prep_and_load(tbl: TargetTableDetails,
                   load: Callable[[DBDriver], Optional[int]],
                   load_exception_type: Type[Exception],
                   reset_before_reload: Callable[[], None] = lambda: None) -> MoveResult:
-    logger.info(f"Connecting to database...")
+    logger.info("Connecting to database...")
```

Comment:
prefer single quotes maybe?

Reply:
I've never gotten a good call on which to use and don't care much but... preserving the formatting here and also this resource recommends that, generally, use doubles for sentences because you might add a contraction, i.e. `"You've got an error!"` and bork your single quotes 🤷 :
https://docs.ckan.org/en/ckan-2.1.5/python-coding-standards.html#use-single-quotes